Source/URLRequest+Alamofire.swift

```diff
@@ -24,9 +24,9 @@
 
 import Foundation
 
-extension URLRequest {
+public extension URLRequest {
     var method: HTTPMethod? {
-        guard let httpMethod = self.httpMethod else { return nil }
+        guard let httpMethod = httpMethod else { return nil }
         return HTTPMethod(rawValue: httpMethod)
     }
 }
```

Comment:
Are we sure we don't want to hide this in `AlamofireExtension` as well?

Reply:
Since we own the `HTTPMethod` type, I left it out, as usage can be disambiguated at the use site if there's a collision. We could certainly put everything inside, it's just makes it more awkward.

Reply:
Yep, makes sense.